src/components/LinkSDK/Lean.js

```diff
@@ -387,29 +401,50 @@ class Lean {
 
   pay({
     account_id,
+    end_user_id,
     show_balances,
     fail_redirect_url,
     payment_intent_id,
     success_redirect_url,
+    bulk_payment_intent_id,
   }) {
-    if (!payment_intent_id) {
-      throw new Error('Validation Error: payment_intent_id is required');
+    if (!payment_intent_id && !bulk_payment_intent_id) {
+      throw new Error(
+        'Validation Error: payment_intent_id or bulk_payment_intent_id is required',
```

Comment:
Do you want to keep validation in LinkSDK only?

Reply:
All basic validations are done in the wrappers to prevent the round network trip. This is usually only triggered for development envs.

Reply:
Ok. I just thought that we should have all validations in one place within LinkSDK web. It's easier to keep that in mind and investigate issues IMO